Firestore/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# Unreleased
+- Add warning when trying to build Firestore's binary SPM distribution for
+  visionOS (#12279). See Firestore's 10.12.0 release note for supported
```

Comment:
super nit: for **a** supported workaround

Reply:
Fixed in 683a33a!